example/index.html

```diff
@@ -92,6 +92,9 @@
 
   <script src='/cms.js'></script>
   <script>
+    var createClass = CMS.createClass;
+    var h = CMS.h;
```

Comment:
I'm not sure if aliasing like this as an example in the docs is a good idea or not. I've done it in the "Customization" doc, but not the "Custom Widgets" doc. Any thoughts?